orchestrator/channels/discord_adapter.py

```
@@ -0,0 +1,159 @@
+        try:
+            import requests
+            token = self.config.get("bot_token", "")
+            resp = requests.get(
+                "https://discord.com/api/v10/users/@me",
+                headers={"Authorization": f"Bot {token}"},
+                timeout=10,
+            )
+            if resp.status_code == 200:
+                user = resp.json()
+                return {"status": "connected", "bot_name": user.get("username")}
+            return {"status": "error", "detail": f"HTTP {resp.status_code}"}
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)